Blueprint.Api.Data/Enumerations.cs

```diff
@@ -19,6 +19,12 @@ public enum IntegrationType {
         Connect = 30
     }
 
+    public enum EventType {
+        Inject = 10,

```

Comment:
Shouldn't we have a type for zero here since that's the DB defautl value for this field?

Reply:
We want to force the user to select one of those, instead of defaulting to one of them.  If we add a 0 value item, we would have to exclude it from the list of selections or change it's name, because spaces aren't allowed in the name (i.e. Choose an inject type).  This might be a good discussion for next Thursday's Tech Talk.